content/manuals/desktop/setup/sign-in.md

```diff
@@ -91,14 +91,32 @@ When Docker CLI or Docker Desktop use credentials, a user prompt may pop up for
 $ docker pull molly/privateimage
 Using default tag: latest
 latest: Pulling from molly/privateimage
-3b9cc81c3203: Pull complete 
+3b9cc81c3203: Pull complete
 Digest: sha256:3c6b73ce467f04d4897d7a7439782721fd28ec9bf62ea2ad9e81a5fb7fb3ff96
 Status: Downloaded newer image for molly/privateimage:latest
 docker.io/molly/privateimage:latest
 ```
 
+## Why might I be required to sign in?
```

Comment:
nit: Can we simplify this heading? Something on the lines of "Why do I need to sign in?"